pos_product_available_negative/models.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    negative_order_group_id = fields.Many2one('res.groups', string='Negative Order Group',
+                                              help='Group allows to sell products which are out of a stock.')
+
+
+class pos_order(models.Model):
+    _inherit = "pos.order"
+
+    supervisor = fields.Many2one('res.users', 'Supervisor',
```

Comment:
good idea, but name it in more unique was, say negative_stock_user_id - "Negative stock approve"